README.md

```diff
@@ -59,6 +59,12 @@ publishing {
 }
 ```
 
+### Spotless
+
+The `net.neoforged.gradleutils.spotless` plugin provides integration (a formatter configuration) with Spotless.  
```

Comment:
Add a note that the actual Spotless plugin must be applied by the buildscript, and that our integration plugin will not pull down (via dependency) or apply the plugin automatically.